docs/docs/setting-up-ci-cd.mdx

```diff
@@ -38,20 +38,22 @@ you can make a test run only if the pull request has a certain label (e.g. “NL
 
 ### Validating Data and Stories
 
-Data validation verifies that there are no mistakes or major inconsistencies in your domain, NLU 
-data, or conversation data. To validate your data, have your CI run this command:
+Data validation verifies that no mistakes or major inconsistencies appear in your domain, NLU 
+data, or story data. To validate your data, have your CI run this command:
 
 ```bash
 rasa data validate --fail-on-warnings --max-history <max_history>
 ```
 
-If you pass a `max_history` value to a Memoization policy in your `config.yml` file, provide the 
-same value in the above validator command. Otherwise, provide the default value of `5`.
+If you pass a `max_history` value to one or more policies in your `config.yml` file, provide the 
+smallest of those values in the above validator command. Otherwise, drop the `--max-history` flag.
 
-If data validation results in errors, training a model will also fail, so it's
+If data validation results in errors, training a model can also fail or yield bad performance, so it's
 always good to run this check before training a model. By including the
 `--fail-on-warnings` flag, this step will fail on warnings indicating more minor issues.
 
+Note: Running `rasa data validate` does **not** test if your [rules](./rules.mdx) are consistent with your stories. 
```

Comment:
Same here: it'd be good to assure the reader that rule-story validation in Rasa is still a thing.